guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+---
+layout: guide
+title: Design best practices
+description: An overview on best design practices for bitcoin-backed ecash.
+nav_order: 5
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/design-best-practices
+redirect_from:
+ - /guide/how-it-works/ecash/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/best-practicies.jpg
+image_base: /assets/images/guide/how-it-works/ecash/
+images_mint-list:
+    - file: federation-list
+      modalImage: federation-list@2x
+      alt: List of federations with QR codes for easy scanning and joining.
+      caption: Federations with QR codes. Each QR code contains the federation's URL, allowing users to display, and invite another person to scan and join.
+    - file: cashu-mint-list
+      modalImage: cashu-mint-list@2x
+      alt: List of Cashu mints with mint addresses displayed.
+      caption: Cashu mints with mint addresses displayed.
+
+images_pending:
+    - file: pending-token
+      modalImage: pending-token@2x
+      alt: A transaction history that shows received, pending, and sent transaction each with their own visual indicator.
+      caption: Received, pending, and sent transaction each with their own visual indicator.
+    - file: pending-transaction-details
+      modalImage: pending-transaction-details@2x
+      alt: Expanded view of a pending transaction, showing the memo, amount, and the mint that created the token.
+      caption: Expanded view of a pending transaction. While token is pending, anyone with the token string can claim it.
+    - file: pending-transaction-qr
+      modalImage: pending-transaction-qr@2x
+      alt: A QR code for a pending transaction with the token string available to copy to clipboard.
+      caption: QR code for a pending transaction.
+
+images_mint-default:
+    - file: cashu-mint-details
+      modalImage: cashu-mint-details@2x
+      alt: Cashu Mint metadata fields displayed in a wallet interface.
```

Comment:
IIRC, it’s not possible to know that information at the mint level. ‘Last connected’ isn’t very helpful, in my opinion, because it only shows the last time your wallet checked the mint’s status. The mint could have been offline for months, but if your wallet checked yesterday and noticed it was offline, that wouldn’t provide much insight. You can only determine whether it’s currently online or offline. With Fedimint, you can see when a Guardian was last online, but with Cashu, you can’t.

You’re right that we should show what we do know, so I added status indicators to display whether a mint is online or offline.